source_control/git.py

```diff
@@ -351,7 +351,7 @@ def has_local_mods(module, git_path, dest, bare):
     cmd = "%s status -s" % (git_path)
```

Comment:
@martin-v could you check if it would help to use `cmd = "%s status --porcelain" % (git_path)` without your previous fix?